scripts/release.ts

```diff
@@ -108,17 +135,44 @@ const doRegularRelease = async () => {
   await git().checkout(['-B', 'release'])
   console.log(chalk.green('Force pushing release branch...'))
   await git().push(['--force', 'origin', 'release'])
-  await createDraftPR()
+  await createDraftRegularPR()
   exit()
 }
 
-const doHotfixRelease = () => {
-  exit(chalk.yellow('Unimplemented. PRs welcome!'))
-  // TODO(0xdef1cafe): implement hotfix release
-  // 1. ask if we want to merge currently checked out branch to main
-  // 2. set release to current branch
-  // 3. force push release to origin
-  // 4. create draft PR
+const doHotfixRelease = async () => {
+  // Only continue if the branch is cleanly branched off origin/main since we will
+  // target it in the hotfix PR
```

Comment:
Just paranoia - not a *big* deal for regular releases since we diff the whole origin/develop commit history against main by resetting things out, but obviously a much bigger issue here if not clean